src/main/java/com/iridium/iridiumskyblock/listeners/PlayerInteractListener.java

```diff
@@ -97,4 +112,13 @@ public void onPlayerInteractEntity(PlayerInteractEntityEvent event) {
         player.sendMessage(StringUtils.color(IridiumSkyblock.getInstance().getMessages().cannotInteractEntities.replace("%prefix%", IridiumSkyblock.getInstance().getConfiguration().prefix)));
     }
 
+    private boolean hasNoCooldown(Player player) {
+        if (cooldownProvider.isOnCooldown(player)) {
```

Comment:
I think we should set the cooldown to like 0.1 seconds then apply the cooldown even if they are already on the cooldown.

So then it will only send the message once until they step of the pressure plate

lmk if this doesnt make sence cuz I dont understand what im saying half the time

Reply:
I thought that they should still know that they are doing something wrong but that's also possible. What would you prefer?

Reply:
id say do it my way